### PR TITLE
Load zxcvbn library from script tag not bundle

### PR DESCRIPTION
**Pull request by eatonphil**
Reduced the pre-gzip size from ~2.1mb to ~1.2mb.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,5 +11,6 @@
     <script src="/static/common.js"></script>
     <script src="/static/bundle.js"></script>
     <script>init();</script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/zxcvbn/4.4.1/zxcvbn.js"></script>
```
Comment (mparke) [inline index.html]: Thoughts on keeping the script separate, but serving from nginx?

**Comment by briansteffens** [comment]
Why does this change test coverage so much?

**Comment by eatonphil** [comment]
Dunno, seems pretty irrelevant to this PR so it must be coming from somewhere else.

**Comment by eatonphil** [comment]
hurrah!

**Pull request merged**